<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Aria2Options</class>
 <widget class="QDialog" name="Aria2Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>261</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>392</width>
    <height>261</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>392</width>
    <height>261</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Aria2cRemote.qrc">
    <normaloff>:/icon/gear.png</normaloff>:/icon/gear.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_Aria2c">
      <attribute name="title">
       <string>Aria2c</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Remote Host:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_RemoteHost</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_RemoteHost">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Port:</string>
       </property>
       <property name="buddy">
        <cstring>spinBox_Port</cstring>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_Port">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>70</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>6800</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>User name:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_UserName</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_UserName">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_Password</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Password">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_Proxy">
      <attribute name="title">
       <string>Proxy</string>
      </attribute>
      <widget class="QCheckBox" name="checkBox_Enable_Proxy">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>133</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Port:</string>
       </property>
       <property name="buddy">
        <cstring>spinBox_Proxy_Port</cstring>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_Proxy_Port">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>70</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>8080</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>User name:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_Proxy_UserName</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Proxy_UserName">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Server:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_Proxy_Server</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Proxy_Server">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_Proxy_Password</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Proxy_Password">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>130</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_server">
      <attribute name="title">
       <string>Server</string>
      </attribute>
      <widget class="QTreeWidget" name="treeWidget_Server_List">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>351</width>
         <height>141</height>
        </rect>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">Server</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>User</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButton_Servers_Delete">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Servers_Add">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Servers_Modify">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_RemoteHost</tabstop>
  <tabstop>spinBox_Port</tabstop>
  <tabstop>lineEdit_UserName</tabstop>
  <tabstop>lineEdit_Password</tabstop>
  <tabstop>checkBox_Enable_Proxy</tabstop>
  <tabstop>lineEdit_Proxy_Server</tabstop>
  <tabstop>spinBox_Proxy_Port</tabstop>
  <tabstop>lineEdit_Proxy_UserName</tabstop>
  <tabstop>lineEdit_Proxy_Password</tabstop>
  <tabstop>treeWidget_Server_List</tabstop>
  <tabstop>pushButton_Servers_Add</tabstop>
  <tabstop>pushButton_Servers_Delete</tabstop>
  <tabstop>pushButton_Servers_Modify</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="Aria2cRemote.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Aria2Options</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Aria2Options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
